src/api/resources/rest/sample.ts

```diff
@@ -65,12 +65,41 @@ export class SampleResource extends RESTResource<types.Sample | types.Presample>
      *
      * @param sampleId Sample id
      * @param collectionIds Array of collection ids
-     * @returns {Rx.Observable<void>}
      */
     public addToCollections(sampleId: number, collectionIds: number[]): Rx.Observable<void> {
         return this.callMethod<void>(sampleId, 'add_to_collection', { ids: collectionIds });
     }
 
+    /**
+     * Remove sample from collections.
+     *
+     * @param sampleId Sample id
+     * @param collectionIds Array of collection ids
+     */
+    public removeToCollections(sampleId: number, collectionIds: number[]): Rx.Observable<void> {
+        return this.callMethod<void>(sampleId, 'remove_to_collection', { ids: collectionIds });
```

Comment:
This is probably wrong as it should be `remove_from_collection`?

Reply:
https://github.com/genialis/resolwe/blob/master/resolwe/flow/views/entity.py#L103

Reply:
Fixed.